backend/de.metas.business/src/main/java/de/metas/order/impl/OrderBL.java

```diff
@@ -1245,4 +1242,18 @@ public Set<OrderAndLineId> getSOLineIdsByPOLineId(@NonNull OrderAndLineId purcha
 		return orderDAO.getSOLineIdsByPOLineId(purchaseOrderLineId);
 	}
 
+	@Override
+	public boolean isOnConsignmentOrder(@NonNull final I_C_Order order)
```

Comment:
this method name is misleading. why not calling it `computeIsOnConsignmentFromLines`?